service/src/main/kotlin/no/nav/su/se/bakover/service/søknadsbehandling/SøknadsbehandlingServiceImpl.kt

```diff
@@ -295,6 +286,8 @@ internal class SøknadsbehandlingServiceImpl(
             IverksettStatusovergangFeilMapper.map(it)
         }.map {
             søknadsbehandlingRepo.lagre(it)
+            // TODO jah, jm: Flytt inn i FerdigstillIverksetting...
```

Comment:
Bør gå igjennom denne flyten en gang og se til at vi håndterer ting slik vi ønsker, dvs at vi lagrer og publiserer statistikk på riktig tidspunkt etc.

Reply:
Må se på forskjell mellom innvilget/avslag her. I mine øyne hadde det ikke vært dumt om vi kjørte samme flyt for avslag og innvilgelse.